src/ccache.cpp

```diff
@@ -86,6 +86,8 @@ Common options:
                               (normally not needed as this is done
                               automatically)
     -C, --clear               clear the cache completely (except configuration)
+    -e, --evict-older-than N  delete files older than N seconds (this will not
```

Comment:
I don't think we should dedicate a short option to this since it's a very narrow use case in my mind, so I suggest going with just `--evict-older-than`.

Reply:
ok.